src/Base/PyObjectBase.cpp

```diff
@@ -96,7 +96,7 @@ PyObjectBase::~PyObjectBase()
  * To prevent subclasses of PyTypeObject to be subclassed in Python we should remove
  * the Py_TPFLAGS_BASETYPE flag. For example, the classes App::VectorPy and App::MatrixPy
  * have removed this flag and its Python proxies App.Vector and App.Matrix cannot be subclassed.
- * In case we want to allow to derive from subclasses of PyTypeObject in Python
+ * In case we want to allow one to derive from subclasses of PyTypeObject in Python

```

Comment:
Maybe:
```suggestion
 * In case we want to allow a new class to derive from subclasses of PyTypeObject in Python
```
Could also go with "developers" or something like that. "One" feels out of place here.